Explorer/Assets/DCL/Landscape/SatelliteFloor.cs

```
@@ -33,10 +35,14 @@ public void Initialize(LandscapeData config)
             materialPropertyBlock = new MaterialPropertyBlock();
 
             satelliteRenderers = new Renderer[SATELLITE_MAP_RESOLUTION * SATELLITE_MAP_RESOLUTION];
+
+            isInitialized = true;
         }
 
         public void Create(MapRendererTextureContainer textureContainer)
         {
+            if (!isInitialized) return;
```

Should we throw an error here? It doesn't create the floor in this case

---

Or just to rename TryCreate